ebusd/DOCS.md

```
@@ -44,8 +44,10 @@ After TCP clients activation you can connect from any system with installed [ebu
 
 The following example will force a reading of all messages from loaded csv config files and can be included via crontab for regular message updates:
 
-```ebusctl -s X.X.X.X f -l "*" -a|awk '{print $2}' | xargs -L1 -t ebusctl -s X.X.X.X r```
-Where ```X.X.X.X``` is the address of the ebusd add-on.
+`ebusctl -s X.X.X.X f -l "*" -a|awk '{print $2}' | xargs -L1 -t ebusctl -s X.X.X.X r`
+
+
```

Markdownlint (0.37.0)

Expected: 1; Actual: 2
Multiple consecutive blank lines

(MD012, no-multiple-blanks)